dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -217,6 +219,44 @@ utilisationReportsRouter
   .route('/:id/selected-fee-records-details')
   .get(validation.sqlIdValidation('id'), handleExpressValidatorResult, getSelectedFeeRecordDetails);
 
+/**
+ * @openapi
+ * /utilisation-reports/:reportId/payment/:paymentId:
+ *   delete:
+ *     summary: Delete a payment
+ *     tags: [Utilisation Report]
+ *     description: Deletes a payment
+ *     parameters:
+ *       - in: path
+ *         name: reportId
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: the id for the report the payment belongs to
+ *       - in: path
+ *         name: paymentId
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: the id for the payment to delete
+ *     responses:
+ *       200:
+ *         description: OK
+ *       404:
+ *         description: Not Found
+ *       500:
+ *         description: Internal Server Error
+ */
+utilisationReportsRouter
+  .route('/:reportId/payments/:paymentId')
+  .delete(
```

Comment:
some lovely API validation in the middleware here, not the first time but just a shout out that's it's great we've introduced more of this!